### PR TITLE
Use log_decorator to do logging

### PR DESCRIPTION
**Pull request by hsong-rh**
#23

### CODE REVIEW

**Comment by hsong-rh** [comment]
@roliveri @jerryk55 Please review.

File: Gemfile [hunk 1]
```diff
@@ -3,7 +3,9 @@ source 'https://rubygems.org'
 # Specify your gem's dependencies in amazon_ssa_support.gemspec
 gemspec
 
+gem "log_decorator", ">0", :require => 'log_decorator', :git => "https://github.com/ManageIQ/log_decorator.git", :branch => "master"
```
Comment (roliveri) [inline Gemfile]: `log_decorator` is a published gem, so you should remove the `:git =>` portion.

File: Gemfile [hunk 2]
```diff
@@ -3,7 +3,9 @@ source 'https://rubygems.org'
...
 gem "manageiq-gems-pending", ">0", :require => 'manageiq-gems-pending', :git => "https://github.com/ManageIQ/manageiq-gems-pending.git", :branch => "master"
+gem "manageiq-smartstate", ">0", :require => 'manageiq-smartstate', :git => "https://github.com/ManageIQ/manageiq-smartstate.git", :branch => "master"
```
Comment (roliveri) [inline Gemfile]: `manageiq-smartstate` is a published gem, so you should remove the `:git =>` portion.

File: Gemfile (hunk 1 above)
Comment (hsong-rh) [reply Gemfile]: Changed them accordingly.

**Pull request merged**